### ISSUE TITLE
NTR: Dihydrothymine high in CSF (Lab: cerebrospinal fluid tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8470]
New term request Dihydrothymine high in CSF (elthat_111121061459): Cerebrospinal fluid  UMLS: 1847705

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000212
- label: Elevated CSF dihydrothymine concentration
- parents: [Abnormal CSF metabolite concentration (HP:0025454)](https://hpo.jax.org/app/browse/term/HP:0025454)
- pmid: 
- def: The concentration of dihydrothymine in the cerebrospinal fluid is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8470
- synonyms: Dihydrothymine high in CSF(NONE)
